x/xion/types/webauthn.go

```diff
@@ -57,6 +64,25 @@ func VerifyRegistration(ctx sdktypes.Context, rp *url.URL, contractAddr string,
 		Challenge:        challenge,
 		UserID:           smartContractUser.WebAuthnID(),
 		UserVerification: protocol.VerificationPreferred,
+		RelyingPartyID:   rp.Host,
+		CredParams: []protocol.CredentialParameter{
+			{
+				Type:      protocol.PublicKeyCredentialType,
+				Algorithm: webauthncose.AlgES256K,
+			},
+			{
+				Type:      protocol.PublicKeyCredentialType,
+				Algorithm: webauthncose.AlgES256,
+			},
+			{
+				Type:      protocol.PublicKeyCredentialType,
+				Algorithm: webauthncose.AlgEdDSA,
+			},
+			{
+				Type:      protocol.PublicKeyCredentialType,
```

Comment:
are there other algos we want to support or remove from this list ? or should we add all ? @2xburnt @ash-burnt @edjroz